backend/tests/services/chat/test_task_default_knowledge_bases.py

```
@@ -209,6 +209,115 @@ def _kind_lookup(_, __, kind_type, ___, name):
+            "app.services.chat.task_default_knowledge_bases.DingTalkDocService.is_configured",
+            return_value=False,
+        ):
+            result = build_initial_task_context_refs(db=db, user=user, team=team)
+
+    assert result["context_refs"] == []
+    assert result["knowledge_base_refs"] == []
+    assert result["context_warnings"] == [
+        {
+            "type": "external_document",
+            "reason": "mcp_not_configured",
+            "message": "未开启钉钉 MCP，无法读取钉钉知识",
```

Ruff (0.15.17)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)